internal/receiver/awscontainerinsightreceiver/host/nodeCapacity.go

```diff
@@ -0,0 +1,51 @@
+package host
+
+import (
+	"os"
+
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/mem"
+	"go.uber.org/zap"
+)
+
+const (
+	GoPSUtilProcDirEnv = "HOST_PROC"
+)
+
+type NodeCapacity struct {
+	MemCapacity int64
+	CPUCapacity int64
+	logger      *zap.Logger
+}
+
+func NewNodeCapacity(logger *zap.Logger) *NodeCapacity {
+	if _, err := os.Lstat("/rootfs/proc"); os.IsNotExist(err) {
+		panic("/rootfs/proc doesn't exists")
```

Comment:
Can we return error instead of panic here

Reply:
Sure.